package.json

```diff
@@ -71,7 +72,7 @@
     "commander": "^2.8.1",
     "css-loader": "^0.15.1",
     "d3": "^3.5.6",
-    "elasticsearch": "^5.0.0",
+    "elasticsearch": "elastic/elasticsearch-js#master",
     "elasticsearch-browser": "^5.0.0",
     "expiry-js": "^0.1.7",
     "exports-loader": "^0.6.2",
```

Comment:
As of last week, plugins are responsible for providing their own dependencies and Kibana doesn't try to install anything. I think we should just move these up to the main package.json.